test/functional/apps/getting_started/_shakespeare.ts

```diff
@@ -228,8 +245,14 @@ export default function ({ getService, getPageObjects }) {
       await PageObjects.visEditor.clickGo();
 
       // same values as previous test except scaled down by the 50 for Y-Axis min
-      const expectedChartValues = [21, 15, 12, 5, 5];
-      const expectedChartValues2 = [127, 56, 103, 82, 112];
+      const expectedChartValues = await PageObjects.visChart.getExpectedValue(
+        [21, 15, 12, 5, 5],
+        [71, 65, 62, 55, 55] // no scaled values in elastic-charts
+      );
```

Comment:
Yeah this test is a little sketchy, and has since been `.skip`'d.

These scaled values are different in vislib than in elastic-charts because vislib is determining the values from the scaled heights of the svg elements and so the values are different. I don't think it is worth it to add this to the debug state.